docs/how_to_add_packages.md

```diff
@@ -149,12 +151,14 @@ If your sources are on GitHub, you can copy the link of the "Download ZIP" locat
 
 Then in your `conanfile.py` method, it has to be used to download the sources:
 
-```
+```py
  def source(self):
-     tools.get(**self.conan_data["sources"][self.version])
+     tools.get(**self.conan_data["sources"][self.version], strip_root=True, destination=self._source_subfolder)
 ```
 
-### How to provide a good recipe
+> :warning: Please be aware that `conan-center-index` only builds from sources and pre-compiled binaries are not acceptable. For more information see our [packaging policy](https://github.com/conan-io/conan-center-index/blob/master/docs/packaging_policy.md).
```

Comment:
There are recipes that package binaries in this repo: msys2 and zulu-openjdk.
Perhaps this line should state that cci tries to build from source as much as possible and that packaging binaries is discouraged.

Reply:
#5792 is a recent example of this too